README.md

```diff
@@ -23,6 +35,38 @@ The Azure Devops pipeline is configured to install prerequisites, run the tests,
 
 When a test fails in the pipeline, Playwright is configured to retry it and record a trace - this trace can be analysed to see the entire flow of the test including the full dom before and after each Playwright action. See [Playwright Trace docs](https://playwright.dev/docs/trace-viewer) for information on how to view the trace.
 
-## Common problems
+## Page Object Model pattern
+
+This exemplar showcases a Page Object Model pattern. Good use of this pattern will give maximum reuse of components between different tests whilst making the tests simple to read and understand.
+
+### Structure
+
+Within the tests folder there is a sub folder for each type of test and a sub folder for the page object model. The page object model is shared between the tests.
+
+The `page-object-model` folder contains a file for each web page - within each file there is a `Page` class that is responsible for interacting with the page and a `PageAssertions` class which is responsible for asserting behaviour of a page.
```

Comment:
How should we deal with elements (e.g. an interactive header) that appears on multiple pages?  Should that have its own POM?

Reply:
This is on my list of things to do and show in this exemplar in the future.